--- prnt/hpcups/HPCupsFilter.cpp.orig	2012-09-18 15:45:46.546793734 +0900
+++ prnt/hpcups/HPCupsFilter.cpp	2012-09-18 15:46:08.966793734 +0900
@@ -408,7 +408,7 @@
     }

	string strPrinterURI="" ,strPrinterName= "";
-	m_DBusComm.initDBusComm(DBUS_PATH,DBUS_INTERFACE, getenv("DEVICE_URI"), m_JA.printer_name);
+	//m_DBusComm.initDBusComm(DBUS_PATH,DBUS_INTERFACE, getenv("DEVICE_URI"), m_JA.printer_name);

     ptr = strstr(m_argv[5], "job-uuid");
     if (ptr) {
@@ -429,7 +429,7 @@
         {
             fputs ("STATE: +hplip.plugin-error\n", stderr);

-			m_DBusComm.sendEvent(EVENT_PRINT_FAILED_MISSING_PLUGIN, "Plugin missing", m_JA.job_id, m_JA.user_name);
+			//m_DBusComm.sendEvent(EVENT_PRINT_FAILED_MISSING_PLUGIN, "Plugin missing", m_JA.job_id, m_JA.user_name);

         }
         dbglog ("m_Job initialization failed with error = %d", err);
